components/core/src/crypto/keys/mod.rs

```diff
@@ -273,6 +273,16 @@ where
             }
         };
 
+        // Ignore hidden files such as .DS_store
+        if dir_entry
+            .file_name()
+            .to_str()
+            .map(|s| s.starts_with("."))
```

Comment:
Given that we have all the key name formatting information already expressed in code, I think that leveraging that code here makes more sense than carving out exceptions. There can be many other exceptions we might need to add later, for instance. 

Ensuring that there is one place where "what it means to be a key file" is represented will make the code easier to reason about and modify going forward.